content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -184,6 +184,27 @@ DaemonSet, `/var/lib/kubelet/pod-resources` must be mounted as a
 
 Support for the "PodResources service" requires `KubeletPodResources` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. It is enabled by default starting with Kubernetes 1.15.
 
+## Device Plugin integration with the Topology Manager
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+The Topology Manager is a new Kubelet component that allows resources to be co-ordintated in a Topology aligned manner. In order to do this, the Device Plugin API was extended to include a `TopologyInfo` struct. 
+
+```gRPC
+message TopologyInfo {
+	repeated NUMANode nodes = 1;
+}
+
+message NUMANode {
+    int64 ID = 1;
+}
+```
+Device Plugins that wish to leverage the Topology Manager can send back a populated TopologyInfo struct as part of the device registration, along with the device IDs and the health of the device. The device manager will then use this information to consult with the Topology Manager to make resource assingment decisions based on Topology alignment.
```

Comment:
```suggestion
Device Plugins can send back a populated `TopologyInfo` struct as part of the device registration, along with the device IDs and the health of the device. The kubelet uses that information to guide resource assignment decisions.
```

Reply:
It is device manager in kubelet who uses the information, so I would prefer to leave it as that. I think kubelet is too vague,